lib/internal/test_runner/mock/mock_timers.js

```diff
@@ -112,7 +183,8 @@ class MockTimers {
         emitter.emit('data', { __proto__: null, aborted: true, reason });
       };
 
-      kResistStopPropagation ??= require('internal/event_target').kResistStopPropagation;
+      kResistStopPropagation ??=
+        require('internal/event_target').kResistStopPropagation;
```

Comment:
if this line wasn't changed, pls rollback to the prev format (just to keep try of what you actually added)

Reply:
I didn't change it but every time I run the linter fix it'll replace the line with a break on both cases